web/packages/teleterm/src/ui/services/keyboardShortcuts/keyboardShortcutsService.ts

```diff
@@ -75,6 +75,28 @@ export class KeyboardShortcutsService {
     return this.shortcutsConfig;
   }
 
+  /**
+   * Some actions can get assigned the same accelerators.
+   * This method returns them.
+   */
+  getDuplicatedAccelerators():
+    | Record<string, KeyboardShortcutAction[]>
+    | undefined {
+    const duplicates = Array.from(this.acceleratorsToActions.entries())
+      .filter(([, shortcuts]) => shortcuts.length > 1)
+      .reduce<Record<string, KeyboardShortcutAction[]>>(
+        (accumulator, [accelerator, actions]) => {
+          accumulator[accelerator] = actions;
+          return accumulator;
+        },
+        {}
+      );
+
+    if (Object.keys(duplicates).length) {
+      return duplicates;
+    }
```

Comment:
Small nit but in general I feel like when returning an array or object, it's better to always return the given type and then at the call site check if it's empty rather than returning undefined if array/object is empty.

This way if the callsite simply wants to map over the array or object, it can do so without checking for undefined as an empty array or an empty object are natural default values for those types.

I feel like this is less common in JS simply because the standard library in JS has no good method for checking if an array or an object is empty.

Reply:
Done